### PR TITLE
Roccat.Swarm version 1.9438

### PR DESCRIPTION
**Pull request by CanePlayz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99221)

### CODE REVIEW

**Comment by CanePlayz** [comment]
This will fail because it can't run unattended. I need some help to figure out what the silent switch for this one is. Running /? gives me this, but it doesn't quite work:

image

**Comment by CanePlayz** [comment]
This comment is just there for the tags to change. You can refer to my earlier comment.

File: manifests/r/Roccat/Swarm/1.9438/Roccat.Swarm.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Roccat.Swarm
+PackageVersion: 1.9438
+Installers:
+- InstallerUrl: https://api.tb-acp.com/device/Support/Software/Download/264/ROCCAT-Swarm.zip
+  Architecture: neutral
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: ROCCAT Swarm_v19438.exe
+  InstallerSha256: 0E111B7422DC224F8C099EB0B972D19A00C69F43D7405A8075F53B8D7D833FA0
```
Comment (Trenly) [inline manifests/r/Roccat/Swarm/1.9438/Roccat.Swarm.installer.yaml:13]:
```suggestion
  InstallerSha256: 0E111B7422DC224F8C099EB0B972D19A00C69F43D7405A8075F53B8D7D833FA0
  InstallerSwitches:
    - Silent: /S /v/qn
    - SilentWithProgress: /S /v/qb
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by CanePlayz** [comment]
@Trenly Thanks!

**Comment by CanePlayz** [comment]
Uhm I am confused

**Comment by CanePlayz** [comment]
@Trenly Didn't work unfortunately

**Comment by stephengillie** [comment]
Related: 
* https://github.com/microsoft/winget-cli/issues/910

**Comment by CanePlayz** [comment]
I'm still in contact with the developers

**Comment by CanePlayz** [comment]
Closing this because devs "couldn't find a way to support a silent installation". :(

**Pull request closed without merging**